pkg/result/processors/invalid_issue.go

```diff
@@ -0,0 +1,54 @@
+package processors
+
+import (
+	"path/filepath"
+
+	"github.com/golangci/golangci-lint/pkg/logutils"
+	"github.com/golangci/golangci-lint/pkg/result"
+)
+
+var _ Processor = InvalidIssue{}
+
+type InvalidIssue struct {
+	log logutils.Log
+}
+
+func NewInvalidIssue(log logutils.Log) *InvalidIssue {
+	return &InvalidIssue{log: log}
+}
+
+func (p InvalidIssue) Process(issues []result.Issue) ([]result.Issue, error) {
+	return filterIssuesErr(issues, p.shouldPassIssue)
+}
+
+func (p InvalidIssue) Name() string {
+	return "invalid_issue"
+}
+
+func (p InvalidIssue) Finish() {}
+
+func (p InvalidIssue) shouldPassIssue(issue *result.Issue) (bool, error) {
+	if issue.FromLinter == "typecheck" {
+		return true, nil
+	}
+
+	if issue.FilePath() == "" {
+		// contextcheck has a known bug https://github.com/kkHAIKE/contextcheck/issues/21
+		if issue.FromLinter != "contextcheck" {
+			p.log.Warnf("no file path for the issue: probably a bug inside the linter %q: %#v", issue.FromLinter, issue)
```

Comment:
Perhaps we should show this message as Info, as there is nothing the user can do in this situation.

Reply:
They can report the issue.